compiler/src/iree/compiler/Codegen/Common/MaterializeUserConfigs.cpp

```diff
@@ -58,9 +87,15 @@ struct MaterializeUserConfigsPass
         getAllEntryPoints(moduleOp);
     MLIRContext *context = moduleOp.getContext();
 
+    bool hasTransformLibrary =
```

Comment:
I think this needs a better explanation of what the different approaches are....